<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>Widget</class>
<widget class="QWidget" name="Widget">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>920</width>
<height>647</height>
</rect>
</property>
<property name="windowTitle">
<string>Widget</string>
</property>
<widget class="QPushButton" name="btn_image">
<property name="geometry">
<rect>
<x>480</x>
<y>560</y>
<width>221</width>
<height>41</height>
</rect>
</property>
<property name="text">
<string>Choose an Image</string>
</property>
</widget>
<widget class="QLabel" name="lbl_image">
<property name="geometry">
<rect>
<x>110</x>
<y>90</y>
<width>300</width>
<height>300</height>
</rect>
</property>
<property name="frameShape">
<enum>QFrame::NoFrame</enum>
</property>
<property name="text">
<string/>
</property>
</widget>
<widget class="QPushButton" name="save_image">
<property name="geometry">
<rect>
<x>40</x>
<y>560</y>
<width>171</width>
<height>41</height>
</rect>
</property>
<property name="text">
<string>Save Image</string>
</property>
</widget>
<widget class="QLabel" name="label">
<property name="geometry">
<rect>
<x>470</x>
<y>460</y>
<width>191</width>
<height>20</height>
</rect>
</property>
<property name="text">
<string>Adjust Brightness</string>
</property>
</widget>
<widget class="QSlider" name="brightnessSlider">
<property name="geometry">
<rect>
<x>430</x>
<y>500</y>
<width>331</width>
<height>22</height>
</rect>
</property>
<property name="maximum">
<number>200</number>
</property>
<property name="pageStep">
<number>1</number>
</property>
<property name="value">
<number>100</number>
</property>
<property name="sliderPosition">
<number>100</number>
</property>
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
</widget>
<widget class="QSlider" name="contrastSlider">
<property name="geometry">
<rect>
<x>50</x>
<y>500</y>
<width>321</width>
<height>22</height>
</rect>
</property>
<property name="maximum">
<number>100</number>
</property>
<property name="sliderPosition">
<number>50</number>
</property>
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
</widget>
<widget class="QLabel" name="label_2">
<property name="geometry">
<rect>
<x>60</x>
<y>460</y>
<width>301</width>
<height>16</height>
</rect>
</property>
<property name="text">
<string>Adjust Contrast</string>
</property>
</widget>
<widget class="QLabel" name="lbl_image_2">
<property name="geometry">
<rect>
<x>440</x>
<y>90</y>
<width>300</width>
<height>300</height>
</rect>
</property>
<property name="frameShape">
<enum>QFrame::NoFrame</enum>
</property>
<property name="text">
<string/>
</property>
</widget>
<widget class="QLabel" name="label_3">
<property name="geometry">
<rect>
<x>550</x>
<y>30</y>
<width>171</width>
<height>16</height>
</rect>
</property>
<property name="text">
<string>Original Image</string>
</property>
</widget>
<widget class="QLabel" name="label_4">
<property name="geometry">
<rect>
<x>170</x>
<y>30</y>
<width>171</width>
<height>16</height>
</rect>
</property>
<property name="text">
<string>Modified Image</string>
</property>
</widget>
</widget>
<layoutdefault spacing="6" margin="11"/>
<tabstops>
<tabstop>btn_image</tabstop>
<tabstop>save_image</tabstop>
<tabstop>contrastSlider</tabstop>
<tabstop>brightnessSlider</tabstop>
</tabstops>
<resources/>
<connections/>
</ui>
